firebase-vscode/src/logger-wrapper.ts

```diff
@@ -45,7 +46,11 @@ export function logSetup() {
     // Re-implement file logger call from ../../src/bin/firebase.ts to not bring
     // in the entire firebase.ts file
     const rootFolders = getRootFolders();
-    const filePath = path.join(rootFolders[0], ".firebase", "logs", "vsce-debug.log");
+    // Default to a central path, but write files to a local path if we're in a Firebase directory.
+    let filePath = path.join("~", ".cache", "firebase", "logs", "vsce-debug.log");
```

Comment:
Is it more confusing to have two locations? 

The only real use case of having the logs without firebase.json is for firebase init.

I'm wondering if we can log to the central location always, to not lose the logs after firebase init.

Reply:
I like keeping the .log files located in the project directory when possible - we already write the firebase-debug.log and <emulator>-debug.log's there